egs/swbd/s5c/local/nnet3/multi_condition/run_ivector_common.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash 
+#set -e
+# this script is based on local/nnet3/run_ivector_common.sh
+# but it operates on corrupted training/dev/test data sets
+
+. cmd.sh
+
+stage=1
+foreground_snrs="20:10:15:5:0"
+background_snrs="20:10:15:5:0"
+num_data_reps=1
+clean_data_dir=train_nodup_sp
+iv_dir=exp/nnet3_rvb
+
+set -e
+. cmd.sh
+. ./path.sh
+. ./utils/parse_options.sh
+
+mkdir -p $iv_dir
+train_set=${clean_data_dir}_rvb${num_data_reps}
+
+if [ $stage -le 1 ]; then
+  if [ ! -d "RIRS_NOISES" ]; then
+    # Download the package that includes the real RIRs, simulated RIRs, isotropic noises and point-source noises
+    wget --no-check-certificate http://www.openslr.org/resources/28/rirs_noises.zip
+    unzip rirs_noises.zip
+  fi
+
+  # corrupt the data to generate reverberated data 
```

Comment:
please clarify via a comment that this doesn't do any real computation, it just writes commands in the wav.scp file.